types/src/crypto/ratchet/snapshots/phnxtypes__crypto__ratchet__test__queue_ratchet_serde_json.snap

```diff
@@ -5,80 +5,76 @@ expression: queue_ratchet()
 {
   "sequence_number": 0,
   "secret": {
-    "key": {
-      "secret": [
-        97,
-        98,
-        99,
-        100,
-        101,
-        102,
-        103,
-        104,
-        105,
-        106,
-        107,
-        108,
-        109,
-        110,
-        111,
-        112,
-        113,
-        114,
-        115,
-        116,
-        117,
-        118,
-        119,
-        120,
-        121,
-        122,
-        48,
-        49,
-        50,
-        51,
-        52,
-        53
-      ]
-    }
+    "secret": [
```

Comment:
Is it a problem for stability (the structure of cbor/json changed)? Or should we consider this change as a breaking change? In case we do, maybe we could also exclude the `_phantom` field from serde?